contracts/core/TribeRoles.sol

```diff
@@ -26,13 +26,19 @@ library TribeRoles {
     /// @notice can mint FEI arbitrarily
     bytes32 internal constant MINTER = keccak256("MINTER_ROLE");
 
-    /// @notice Can grant any admin role
+    /// @notice Manages lower level - Admin and Minor - roles. Able to grant and revoke these
     bytes32 internal constant ROLE_ADMIN = keccak256("ROLE_ADMIN");
 
     /*///////////////////////////////////////////////////////////////
                                  Admin Roles
     //////////////////////////////////////////////////////////////*/
 
+    /// @notice has access to all admin functionality on pods
+    bytes32 internal constant POD_ADMIN = keccak256("POD_ADMIN");
+
+    /// @notice capable of granting and revoking other TribeRoles from having veto power over a pod
+    bytes32 internal constant POD_VETO_ADMIN = keccak256("POD_VETO_ADMIN");
```

Comment:
Just a note, this will be granted to nope dao?

Reply:
Yep, that's the plan